index.html

```diff
@@ -5,20 +5,58 @@
   <title>Title</title>
   <link rel="stylesheet" href="css/main.css">
   <link rel="stylesheet" href="css/card.css">
+  <link rel="stylesheet" href="css/modal.css">
 </head>
 <body>
-  <div id='root'>
-    <header>
-    </header>
-    <main>
-      <div class='container'>
-        <ul class='items'>
-        </ul>
-      </div>
-    </main>
-    <footer></footer>
-  </div>
+<div id='root'>
+  <main>
+    <div class='container'>
+      <ul class='items'>
+      </ul>
+    </div>
+    <div class="modal">
+      <form class="form modal__form">
+        <div class="form__container form__column">
 
-  <script src='js/index.js' type='module'></script>
+          <label for="deviceModel" class="form__label">Enter device model:<span class="modal__required"> * </span></label>
```

Comment:
Заменить этот непонятный спан, на модификатор с псевдоэлементом
```suggestion
          <label for="deviceModel" class="form__label form_label--required">Enter device model:</label>
```